rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -11,17 +11,24 @@ if [ -z `getent passwd spinnaker` ]; then
   useradd --gid spinnaker spinnaker -m --home-dir /home/spinnaker
 fi
 
+create_temp_dir() {
+  PATH=$PATH:/usr/local/bin
```

Comment:
helm installation fails without this.

Reply:
good find. 

Small nit that it isn't `temp_dir` related and is being treated as a global so could go at top of file.
I always find bash weird how you can define a variable inside a function without `local` and it becomes a global.